Aimbot

ManiacMoustache opened the pull request:
Relies on DoofConfigUpdate

ManiacMoustache committed:
Flake8
First of many flake8 commits

DrKittyKat committed:
Added doc strings and notes

cannedlock committed:
added tunables

DrKittyKat committed:
Static analysis changes

DrKittyKat committed:
Registed the 'x' button in robot.py to enable the solenoid on push and disable on release

cannedlock committed:
finished tunables

cannedlock committed:
fixed sensorObjects

ManiacMoustache committed:
Switched from using bad method
Used Xboxmap.getmechY(), that method no longer exist. Now used a
different variable

ChrisHirsch committed:
Merge pull request #75 from Raptacon/compressor-and-pneumatics

Compressor and pneumatics

ChrisHirsch committed:
Merge branch 'StateMachine-shooter' of github.com:Raptacon/Robot-2020 into StateMachine-shooter

ChrisHirsch committed:
Moved testBot.py to testSensors and commented code

ChrisHirsch committed:
For PR#73 added back in the comment for the 3200 robot base class

DrKittyKat committed:
Removed self.motors

cannedlock committed:
Merge branch 'StateMachine-shooter' into Doof-buttonmanager-integration

DrKittyKat committed:
Merge branch 'StateMachine-shooter' of https://github.com/Raptacon/Robot-2020 into StateMachine-shooter

ChrisHirsch committed:
Removed the unit tests agian...accidental add

DrKittyKat committed:
Merge branch 'StateMachine-shooter' of https://github.com/Raptacon/Robot-2020 into StateMachine-shooter

ManiacMoustache committed:
camelCasing
camelCased the shooterMotorsVariable

ManiacMoustache committed:
Merge branch 'StateMachine-shooter' of https://github.com/Raptacon/Robot-2020 into StateMachine-shooter

ManiacMoustache committed:
Brakes
Brake mode on spark maxes is now changable through the key 'Brake' and
True or False

ManiacMoustache committed:
Changed brake name
now to change Idle mode, use IdleBrake and True/False in config.yml

tuffinmuffin committed:
Added config searching. Uses file stored in home dir to get name of
config file. If not found fallsback to default

ChrisHirsch committed:
Moved autonomous mode so it's disabled since it woudl crash otherwise

tuffinmuffin committed:
fixed lint findings

tuffinmuffin committed:
Merge branch 'master' into feature-smartconfig

cannedlock committed:
.

cannedlock committed:
Up to date NOW

cannedlock committed:
Merge branch 'StateMachine-shooter' of https://github.com/Raptacon/Robot-2020 into StateMachine-shooter

ManiacMoustache committed:
Changed names
Changed names of joystick. Changed names of ShooterMotors to
shooterMotors for camelCasing reasons

ManiacMoustache committed:
Merge branch 'StateMachine-shooter' of https://github.com/Raptacon/Robot-2020 into StateMachine-shooter
Merging the monster. I don't like it

ChrisHirsch committed:
Merge pull request #78 from Raptacon/feature-smartconfig

Added config searching based on home dir file

tuffinmuffin committed:
updated minibot config to match minibot

ChrisHirsch committed:
Merge branch 'master' into feature-minibot-config

ChrisHirsch committed:
Merge pull request #79 from Raptacon/feature-minibot-config

updated minibot config to match minibot

ChrisHirsch committed:
Merge branch 'StateMachine-shooter' of github.com:Raptacon/Robot-2020 into StateMachine-shooter

ChrisHirsch committed:
Fixed PIP8 linting

ChrisHirsch committed:
Merge branch 'master' of github.com:Raptacon/Robot-2020 into StateMachine-shooter

ChrisHirsch committed:
Broken build...still working through changes...teleop fails

cannedlock committed:
final changes

cannedlock committed:
changes to state_tm

cannedlock committed:
corrested shooterMotors errors

cannedlock committed:
fixed loader values

cannedlock committed:
Fixed minor issues, got dirty (but likely functional) automatic working

cannedlock committed:
removed prints

cannedlock committed:
minor changes made to improve automatic logic

ManiacMoustache committed:
No real changes
Git is seeing changes where there are none

ManiacMoustache committed:
Merge branch 'StateMachine-shooter' of https://github.com/Raptacon/Robot-2020 into StateMachine-shooter
you dont' need brackets

ManiacMoustache committed:
added elev
elev now not tested

ChrisHirsch committed:
State machine shooter (#73)

Everything changed:
Added Falcon 500s- they work
Changed configmapper and config:
New structure for config file and mapper. It works
It's mostly better
* Added shooter

* Added ready to fire

* Improved sensor logic

* Added Sensor Logic 

* Joystick control

* Added button callback for shooter activation

* towerMotors.py clean-up; dummy encoder added

* Changed sensor assignments

* Fixed sensor ports; added controller buttons

* Good

Consolidated motor creation to robotMap

* Mostly working manual/automatic shoot; crashes when more than 4 balls loaded in automatic

* Added elevator to config.yml

* added state_tm

* Added button manager
right now it switches modes
* added tunables

* Switched from using bad method
Used Xboxmap.getmechY(), that method no longer exist. Now used a
different variable

* Moved testBot.py to testSensors and commented code

* camelCasing
camelCased the shooterMotorsVariable

* Brakes
Brake mode on spark maxes is now changable through the key 'Brake' and
True or False

* Changed brake name
now to change Idle mode, use IdleBrake and True/False in config.yml

* changes to state_tm

* corrected shooterMotors errors

* fixed loader values

* Fixed minor issues, got dirty (but likely functional) automatic working

Co-authored-by: ManiacMoustache <43834579+ManiacMoustache@users.noreply.github.com>
Co-authored-by: Nathan Beasley <tuffinmuffin@gmail.com>
Co-authored-by: cannedlock <56943162+cannedlock@users.noreply.github.com>
Co-authored-by: DrKittyKat <43834685+DrKittyKat@users.noreply.github.com>

ManiacMoustache committed:
It work
kind of

DrKittyKat committed:
lifter works
after changing device id on robot

ManiacMoustache committed:
Merge branch 'elevator-Doof' of https://github.com/Raptacon/Robot-2020 into elevator-Doof

ManiacMoustache committed:
amalgam mess
it works

ManiacMoustache committed:
Merge branch 'master' of https://github.com/Raptacon/Robot-2020 into elevator-Doof
it works

ManiacMoustache committed:
Changed channels
Pneumatics work after changing channel values in code

cannedlock committed:
fixed most syntax issues brought up

cannedlock committed:
Merge branch 'master' into fix-syntax

ManiacMoustache committed:
Changed comment
Corrected the comment

cannedlock committed:
small changes

cannedlock committed:
Merge branch 'fix-syntax' of https://github.com/Raptacon/Robot-2020 into fix-syntax

ManiacMoustache committed:
fixed unnecessary var
No more self.motor

cannedlock committed:
fixed multipliers

ManiacMoustache committed:
tunability
made raise and lower functions of elevator a tunable speed.

cannedlock committed:
changed loader values to tunables

cannedlock committed:
fixed syntax

cannedlock committed:
added comments

ChrisHirsch committed:
Merge pull request #81 from Raptacon/fix-syntax

Pull revised shooter into master

ChrisHirsch committed:
Merge branch 'master' into elevator-Doof

ChrisHirsch committed:
Merge pull request #80 from Raptacon/elevator-Doof

Elevator doof

hell0w0rld2 committed:
fixes the autonomous code

hell0w0rld2 committed:
imports drive train

ChrisHirsch committed:
Merge pull request #83 from Raptacon/autonomous-fix

Autonomous fix

tuffinmuffin committed:
Bug fixmotormaps (#84)


* Component mapper is reworked.
Added ability to check compatibility strings in components. THis will
attempt to disable them if not compatible.
Moved motorHelper
Moved motor creation to robot
updated configs for new component mapper
Tested on sim with minibot, scorpion, and doof for drive train
functionality

* Component refactor and change to elevator so that it uses the new motors_loader rather than the old motorsList

* addeed scorpion loader code, very quick
updated scorpion loader mappings

Co-authored-by: DrKittyKat <43834685+DrKittyKat@users.noreply.github.com>

cannedlock committed:
State machine shooter refactor (#85)

Refactored shooter code to be state based.
Added automatic shooter mode.

DrKittyKat committed:
Windows package install script (#87)

* added initial code

* Added robotpy to installer and added requirements files

* Changed working directory and updated gitignore

* Moved into the script folder into "utils" as it is a utility

tuffinmuffin committed:
Improve config (#88)

* Added more config componets
added more factories
configured pneumatics to use new factories

* cleaned up control types
added gyro refernce to drive train

ManiacMoustache committed:
Sensitivity (#86)

* Sensitivity
applies a ^1.6 curve to the right/left drive controller input in
robot.py's initPeriodic

ManiacMoustache committed:
Adds current limit methods
Tested, works with ctre/rev controllers

hell0w0rld2 committed:
starting to do creeper mode

hell0w0rld2 committed:
gets sim to run

cannedlock committed:
Actual state machine shooter, fix to autonomous (#89)

reworking to to a pure shooter/loader state machine

hell0w0rld2 committed:
working creeper mode

ManiacMoustache committed:
Added TalonFX
Now has TalonFX current configs

hell0w0rld2 committed:
merge stuff

hell0w0rld2 committed:
fixes stuff with the merge

ManiacMoustache committed:
Merge branch 'master' of https://github.com/Raptacon/Robot-2020 into current-limits
merged

ManiacMoustache committed:
Fixed
Spark maxs are fiddly. No idea how we fixed them.

ManiacMoustache committed:
Changed out tests
tried testing on robot -works

DrKittyKat committed:
Reset driveMotorsMultiplier to a tunable of .5

hell0w0rld2 committed:
fixes creeper mode

ManiacMoustache committed:
Change channel

From 41 to 40 because we changed spark maxes

ManiacMoustache committed:
Removed ramprate

not being used

hell0w0rld2 committed:
fixes based on comments

ManiacMoustache committed:
fixed names
Names are fixed

ManiacMoustache committed:
Merge branch 'current-limits' of https://github.com/Raptacon/Robot-2020 into current-limits

ChrisHirsch committed:
Fixed autonomous crashing because of an inflight changes to setTank vs setup

ChrisHirsch committed:
Merge pull request #93 from Raptacon/autonmous_fix

Fixed autonomous crashing because of an inflight changes to setTank v…

ChrisHirsch committed:
Merge branch 'master' into creeper-mode

ChrisHirsch committed:
Merge branch 'master' into elevator-fix

ChrisHirsch committed:
Merge branch 'master' into current-limits

ChrisHirsch committed:
Merge pull request #92 from Raptacon/creeper-mode

Creeper mode

ChrisHirsch committed:
Merge branch 'master' into elevator-fix

ManiacMoustache committed:
Merge branch 'master' into current-limits

ChrisHirsch committed:
Merge pull request #91 from Raptacon/elevator-fix

Spark Max Fixes

ManiacMoustache committed:
Documentation
Added comments

ChrisHirsch committed:
Merge branch 'master' into elevator-fix

ManiacMoustache committed:
Merge branch 'master' into current-limits

ChrisHirsch committed:
Merge pull request #94 from Raptacon/elevator-fix

Documentation

ChrisHirsch committed:
Merge branch 'master' into current-limits

DrKittyKat committed:
changed to use tank mode, inverted all motor (#95)

cannedlock committed:
Shooter callbacks (#96)

* New sensor factory for Digital inputs
* Sensor callbacks for detecting events on Digital input values changes.

Co-authored-by: ManiacMoustache <43834579+ManiacMoustache@users.noreply.github.com>
Co-authored-by: hell0w0rld2 <43834630+hell0w0rld2@users.noreply.github.com>
Co-authored-by: Chris Hirsch <chris@base2technology.com>

cannedlock committed:
fix-CRITICAL ERROR, MERGE ASAP (#97)

* first commit

* finished, testing

* more working changes

* Fixed syntax error

* fixed another syntax error

* new sensors ready for testing

* added stuff

* Should be finished, needs sim/bot testing

* needs fixes, mostly done

* fixed some changes, added comments

* More changes, more comments

* minor changes, ready for bot/sim testing

* fixed syntax

* fixed camelCasing

* fixed elevator, ShooterMotors clean-up, added prints to logic

* fixed 'stopShooter()' bug

* fixed spelling error

* Changed logic
Many logic changes

* creats location to add in future values

* more logic changes
even more

* simplified loader logic

* small changes

* small changes

* small changes

* after bot testing, needs fixes

* new stuff to test for shooter

* needs review, motors don't work

* fixed motor issue

* post testing changes

* some flake8 fixes

* sets up shooting in autonomous

* adds tunable

* uncommented compatString

* final commits before merge

* fixed errors

* autonomous stuff

* test shooting in autonomous

* fixed shooter encoder logic

* starts to fix autonomous

* shoots in autonomous

* total refactor of statemachine. should be much cleaner. needs testing

* syntax changes

* fixed shooter syntax error

* fine tuning after testing, ready to merge

* fixed autonomous, needs testing

* fixed more autonomous stuff, added docstrings

* added small changes

* (hopefully) fixed autonomous

* fixed autonomous, now shoots and drives

* final changes to autonomous

* fixed drive speed in autonomous

* added some comments

* basic clean-up

* more basic clean-up

* fixed statemachine-shooter

* fixed sensor enum

* Turns in autnomomous

* fixed names

* fixed errors

* self.engage() constantly called now

* self.engage() constantly called now

* added docstrings

* fixed imports

* changed var names

* changed var names

* added another state for loading

* fixed states

* fixed more name issues

* added enums for direction of motors

* added docstrings

* fixed enum class location

* removed unneccessary imports

* Broken stuff, A LOT of changes, needs significant review

* Added simple (probably stupid) callback example

* Needs significant reworks

* almost there

* finished callbacks, ready for bot testing

* finished callbacks, ready for bot testing

* tuning changes

* nearly done, final working changes

* nearly done, final tuning changes

* added some docstrings

* final changes, syntax clean-up

* flake8 fixes

* flake8 fixes

* flake8 fixes

* changed target shooting speed

* minor number changes

* fixed stuff from nathans comments

* fixed critial error, NEEDS TO MERGE ASAP

* fixed critial error, NEEDS TO MERGE ASAP

Co-authored-by: ManiacMoustache <43834579+ManiacMoustache@users.noreply.github.com>
Co-authored-by: hell0w0rld2 <43834630+hell0w0rld2@users.noreply.github.com>
Co-authored-by: Chris Hirsch <chris@base2technology.com>

tuffinmuffin committed:
fixed creeper double enable losing state

ChrisHirsch committed:
Merge pull request #99 from Raptacon/bug-creepermodeDisableFailure

fixed creeper double enable losing state

Drivede committed:
Added Sample Auto Aim code, likely missing some critical pieces to the reading of network tables.  Majority of code is a state machine to handle full auto-aim then shoot with single button press.

ManiacMoustache committed:
flipped the motor to what it should be

for some reason, the motor flipped it's direction. I suspect it was
hardware, but we fixed it here

ChrisHirsch committed:
Merge pull request #102 from Raptacon/lifter-invert

flipped the motor to what it should be

Drivede committed:
Delete slnx.sqlite

Drivede committed:
Delete ProjectSettings.json

Drivede committed:
Delete PythonSettings.json

Drivede committed:
Delete VSWorkspaceState.json

Drivede committed:
Delete .suo

ManiacMoustache committed:
Merge branch 'master' into current-limits

cannedlock committed:
Actual shooter fix (#107)

*Refactor of sensors and loader / shooter logic

ManiacMoustache committed:
Merge branch 'master' into current-limits

ChrisHirsch committed:
Merge pull request #90 from Raptacon/current-limits

Adds current limit methods

cannedlock committed:
fixed lifter.yml indent error

ChrisHirsch committed:
Merge pull request #111 from Raptacon/bug-yamlParseErr

fixed lifter.yml indent error

tuffinmuffin committed:
Feature strict state machine (#103)

* added StrictStateMachine.
See stroll.py in examples. Various modes can be done by changing the internal logic. Committed version will throw exception going into Invalid

ChrisHirsch committed:
Urgent shooter fixes (#113)

* works now

* fixed flake8

* fixed flake8

* working changes'

* working changes

* nearly done, working changes

* ready for merge

* small changes

* small changes

* I will now flake
for flakyness

* final changes

* changes some breaksensor stuff

* organization

Co-authored-by: cannedlock <zwinter03@gmail.com>
Co-authored-by: ManiacMoustache <43834579+ManiacMoustache@users.noreply.github.com>
Co-authored-by: ManiacMoustache <cjkesler@unionvotes.net>

ManiacMoustache committed:
Lifter to winch naming fix (#108)

Renamed lifter to winch

cannedlock committed:
Fix to master; significant clean-up (#117)

* needs work, not done

* first commit, still needs some work

* almost done, needs testing on bot before merge.

* small changes

* formatting stuff

* bot testing done, ready to merge.

* removed a pring

* fixed some things

* removed some imports

* minor tuning

* more tuning

* final changes

* flake8

Co-authored-by: ManiacMoustache <43834579+ManiacMoustache@users.noreply.github.com>

ManiacMoustache committed:
Currentlimit hopper fix (#116)

* Adjust hopper current limits

ManiacMoustache committed:
Shooter pid (#100)

* Added velocity control to shooter

ManiacMoustache committed:
inverted (#119)

Now the hopper motor runs the right way

ManiacMoustache committed:
Pid fix (#118)

* inverted
Now the hopper motor runs the right way

* Fixed all the things that didn't end up in master
Little stuff like actually setting the moto

* Flake8

* removed prints
no more control type

cannedlock committed:
Integrated PID shooter into ShooterLogic (#121)

* inverted
Now the hopper motor runs the right way

* Fixed all the things that didn't end up in master
Little stuff like actually setting the moto

* Flake8

* removed prints
no more control type

* added different speed for autonomous

* finished fixing logic, needs bot testing

* ready for bot testing

* basic cleanup

* small uncaught changes

* removed unneeded vars

* small variable changes

* changed some vars to make code easier to understand

* changes to fix shooter/loaderlogic

Co-authored-by: ManiacMoustache <43834579+ManiacMoustache@users.noreply.github.com>

ManiacMoustache committed:
Added "aimbot.py" and "rpmToDistance.yml" to start
tuning RPM based on distance

ManiacMoustache committed:
Merge branch 'AutoAimSample' of https://github.com/Raptacon/Robot-2020 into aimbot

ManiacMoustache committed:
made things work (partly) (still making networktables fit)

ManiacMoustache committed:
made things work
(did someone forget to commit fixes to the AutoAim branch at some point?
There are a lot of stupid little things here)

ManiacMoustache committed:
I can't get over that THAT was the issue

ManiacMoustache committed:
Test code (make sure network tables work)
and fix things (use the getNumber() method to get numbers from the network
talbes) Also added the check for a valid target

ManiacMoustache committed:
Not crashing
(Probably isn't working though)

ManiacMoustache committed:
added quadratic section to yml

ManiacMoustache committed:
removed aimbot.py
planning to integrate any rpm changes into autoAim.py

ManiacMoustache committed:
added calculateRPM() and changed findConfig()
findConfig now has an argument for the name of the config. The only time this is
currently used is to find the robot config but I'm planning to use it to
store values for RPM calculations so I added the argument so I could use the
method to find the yml.

ManiacMoustache committed:
changed findConfig() a little
Maybe I could use it? for now i'm making a new function

ManiacMoustache committed:
More tests
ended up copying, pasting and changing. Could cut out some stuff only needed when
different versions are available.

ManiacMoustache committed:
one comment on regressions

ManiacMoustache committed:
added setRPM() func to shooterLogic
figured it was necessary if we're changing rpms. THIS IS STILL TEST CODE,
IT CANNOT CALCULATE DISTANCE.

ManiacMoustache committed:
More test code
This should be ready to test on the robot
First we need to get values for the yml and run a regression.
Then we test.

ManiacMoustache committed:
changed prints to logs

ManiacMoustache committed:
removed test code
should be functional with a controller

ManiacMoustache committed:
removed trailing whitespace

ManiacMoustache committed:
switching to linear curve
Instead of equation, find point behind and in front of current distance
and approximate

ManiacMoustache committed:
added linear curve
calculates a line between two values and what the RPM should be for that
point on the line

ManiacMoustache committed:
Added test code to test limelight

ManiacMoustache committed:
More advanced test code

ManiacMoustache committed:
added test code

ManiacMoustache committed:
added log error

ManiacMoustache committed:
Moved test code

ManiacMoustache committed:
Moved test code again

ManiacMoustache committed:
Moved tests to test function

ManiacMoustache committed:
Commente out some things to test

ManiacMoustache committed:
Added a good question

ManiacMoustache committed:
changed error issues

ManiacMoustache committed:
added init test code

ManiacMoustache committed:
changed IP

ManiacMoustache committed:
Changed the test code

ManiacMoustache committed:
Removed test code

ManiacMoustache committed:
Clarified error

ManiacMoustache committed:
Removed unnecessary indent

ManiacMoustache committed:
Removed unnecessary import

ManiacMoustache committed:
Removed unecessary import 2

ManiacMoustache committed:
final tests of limelight

ManiacMoustache committed:
Merge branch 'aimbot' of https://github.com/Raptacon/Robot-2020 into aimbot

ManiacMoustache committed:
Switched from tunables to network table numbers

ManiacMoustache committed:
Split autoAim into two components (autoAlign and autoShoot) plus an extra util

ManiacMoustache committed:
Cleaned it up for flake

ManiacMoustache committed:
Prevented drivetrain from using controllers during autoAlign and added stops while autoalign isn't running

ManiacMoustache committed:
fixed autoAlign and fixed network table variables for diagnostics

ManiacMoustache committed:
made autoalign PID controlled. Prepare for oscillating drivetrain

ManiacMoustache committed:
cleaned up code

ManiacMoustache committed:
literally one new line

ManiacMoustache committed:
PID aligning and cleaning

Team3200 committed:
removed unncessary line

Team3200 committed:
removed whitespace

Team3200 committed:
updated config

ManiacMoustache committed:
Merge branch 'master' into DoofConfigUpdate

Team3200 committed:
changed break sensor channels

Team3200 committed:
testing changes for break sensors

Team3200 committed:
merged old aimbot branch

Team3200 committed:
added autoaim to robot.py

Team3200 committed:
resolved conflicts and added compatibility

Team3200 committed:
Merge branch 'master' of https://github.com/Raptacon/Robot-2021 into aimbot

Team3200 committed:
changed things

Team3200 committed:
merged DoofConfigUpdate

Team3200 committed:
added newline

Team3200 committed:
added compatibility testing

Team3200 committed:
fixed break sensors

Team3200 committed:
newline

Team3200 committed:
changed one line

Team3200 committed:
Merge branch 'DoofConfigUpdate' of https://github.com/Raptacon/Robot-2021 into DoofConfigUpdate

Team3200 committed:
removed test code

Team3200 committed:
resolved more conflicts

Team3200 committed:
more conflicts

Team3200 committed:
updated ci

Team3200 committed:
ignored rules

Team3200 committed:
ignored rules in correct place

Team3200 committed:
removed unnecessary imports

Team3200 committed:
merged with DoofConfigUpdate

Team3200 committed:
resolved robot conflict

Team3200 committed:
changed shooter state machine

Team3200 committed:
overrode safety features (when I want the hopper to run, it RUNS!)

Team3200 committed:
updated state machine for autoshoot and PID for autoalign

Team3200 committed:
resolved merge conflicts with master

Team3200 committed:
there wasn't a conflict, but ok

Team3200 committed:
updated distance to RPM line code

Team3200 committed:
updated dist to RPM table - it works great!

Team3200 committed:
updated shooterLogic and default RPM

Team3200 committed:
updated autoShoot's default RPM

Team3200 committed:
Merge branch 'master' of https://github.com/Raptacon/Robot-2021 into aimbot

tuffinmuffin commented on .deploy_cfg:
```diff
@@ -0,0 +1,3 @@
+[auth]
+hostname = 10.32.0.2
```
remove file?

tuffinmuffin commented on .installer_config:
```diff
@@ -0,0 +1,3 @@
+[auth]
+hostname = 3200
```
remove file?

tuffinmuffin commented on deploy_utils/.installer_config:
```diff
@@ -0,0 +1,3 @@
+[auth]
+hostname = 3200
+
```
delete?

tuffinmuffin commented on deploy_utils/Installer-script-win/.installer_config:
```diff
@@ -0,0 +1,3 @@
+[auth]
+hostname = 3200
```
delete?

Team3200 committed:
made nice for flake8

Team3200 committed:
motorHelper update - added braking function to all talonFXs. All of them

Team3200 committed:
added 'brakingBound' which controls when the robot starts braking while turning

Team3200 committed:
Revert "added 'brakingBound' which controls when the robot starts braking while turning"

This reverts commit c66ba3996774bd4a282525c18db506cf019b001b.

Team3200 committed:
Revert "added 'brakingBound' which controls when the robot starts braking while turning"

This reverts commit c66ba3996774bd4a282525c18db506cf019b001b.

Team3200 committed:
Merge branch 'aimbot' of https://github.com/Raptacon/Robot-2021 into braking-autoalign

Team3200 committed:
added brakingBound

Team3200 committed:
Revert "made nice for flake8"

This reverts commit 010d44e8f19f2817c05920df8a655e492c5e46e7.

Team3200 committed:
Revert "motorHelper update - added braking function to all talonFXs. All of them"

This reverts commit cd9745f546925b992362828b247d881f40c01ea9.

Team3200 committed:
reverted to old talonFX follower

Team3200 committed:
changed 'setBraking
' method

Team3200 committed:
disabled braking while disabled and changed brakingBound logic

Team3200 committed:
Revert "disabled braking while disabled and changed brakingBound logic"

This reverts commit 4495cb38e861f93ea083cff00528109de1e55ce6.

Team3200 committed:
edited gitignore

Team3200 committed:
removed unnecessary items

Team3200 committed:
merged with aimbot

ManiacMoustache committed:
Merge branch 'master' into aimbot

ChrisHirsch commented on components/shooterLogic.py:
```diff
@@ -1,7 +1,7 @@
 from robotMap import XboxMap
+import logging as log
```
Needed?

ManiacMoustache replied:
Yes, we use it multiple times throughout the file

ChrisHirsch reviewed (commented):
Clean up some code and lets take this branch out for a drive and we're good to go

Team3200 committed:
changed autoalign to be more working

Team3200 committed:
Merge branch 'aimbot' of https://github.com/Raptacon/Robot-2021 into aimbot

Team3200 committed:
Merge branch 'braking-autoalign' of https://github.com/Raptacon/Robot-2021 into aimbot

Team3200 committed:
perfected autoAlign

ManiacMoustache committed:
Delete 2020venv.bat

File shouldn't be tracked

ManiacMoustache committed:
added compat to lidar

ManiacMoustache committed:
merged two drive solutions

ManiacMoustache committed:
Finished merging

ManiacMoustache committed:
Really finished merging

ChrisHirsch committed:
ci: 🎡 ignore ONLY W503 and keep W504

per https://github.com/mila-iqia/cookiecutter-pyml/issues/10 this is
what's now recommended

ChrisHirsch reviewed (approved):
+1 ok lets see how this behaves...there could definitely be some improvements but overall I'm liking the documentation, better comments and the components
